core/src/main/java/org/apache/accumulo/core/security/crypto/CryptoModuleFactory.java

```diff
@@ -239,14 +239,6 @@ public CryptoModuleParameters initializeCipher(CryptoModuleParameters params) {
 
   }
 
-  public static String[] parseCipherTransform(String cipherTransform) {
-    if (cipherTransform == null) {
-      return new String[3];
-    }
-
-    return cipherTransform.split("/");
```

Comment:
This parsing was bad, it did no validation.  It seems like this changes removes all custom parsing and uses java standard libs to do the parsing, is this correct?

Reply:
We leave it to Cipher to parse. 

https://docs.oracle.com/javase/8/docs/api/javax/crypto/Cipher.html#getInstance-java.lang.String-

Reply:
Yeah I noticed the changes to Cipher.getInstance() .  Do you think all of the custom parsing code was removed?  I saw a lot removed in the PR, I wasn't sure if this was all if it though.  Thought it would be quicker to ask than look myself.  If you are not sure, I can look.

Reply:
I don't see any more splits in the crypto code